traitcuration/traits/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+
+
+class Trait(models.Model):
+    name = models.CharField(max_length=200)
+    current_mapping = models.ForeignKey(
+        'Mapping', on_delete=models.SET_NULL, null=True, blank=True)
+    status = models.CharField(max_length=50)
+    number_of_source_records = models.IntegerField(blank=True, null=True)
+    timestamp_added = models.DateTimeField(auto_now=True)
+    timestamp_updated = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.name
+
+
+class User(models.Model):
+    username = models.CharField(max_length=100, unique=True)
+    email = models.EmailField(unique=True)
+    role = models.CharField(max_length=20, blank=True, null=True)
+
+    def __str__(self):
+        return self.username
+
+
+class OntologyTerm(models.Model):
+    curie = models.CharField(max_length=50, blank=True, null=True, unique=True)
+    uri = models.URLField(null=True, blank=True, unique=True)
+    label = models.CharField(max_length=200)
+    status = models.CharField(max_length=50)
```

Comment:
This should also (eventually) be not a char, but some sort of enum

Reply:
It can be converted into an enum by adding a `choices` attribute to the field. I will add that in the implemented changes